src/AtomShell/window.jl

```diff
@@ -30,9 +30,25 @@ function Window(a::Shell, opts::Associative = Dict())
     Window(a, Page(), opts)
 end
 
+default_async = false
```

Comment:
I don't think it's a good idea to have behavior controlled by a mutable global variable like this. It means that a package that tries to set it one way can't be used alongside a package that tries to set it the other way. Why not just mark this `const`? Or, even simpler, just remove this, remove `is_async`, and replace its usages with `get(opts, :async, false)` ?

Reply:
`get(opts, :async, false)` is the best option imho.

Reply:
Oops, I agree. Sorry, i think I only left that in there for testing, so i could switch between them when running all the tests without having to change each one. Sorry!! 🤦‍♂️  I'll remove this. Yes I agree with you both, thanks!

Thanks for taking a look!!

Reply:
I've fixed it in what I pushed up, and now I'm going to rebase after having checked in #136.